crates/m3/src/builder/table.rs

```diff
@@ -420,13 +430,25 @@ impl<'a, F: TowerField> TableBuilder<'a, F> {
 	{
 		assert!(pow_bits.len() <= (1 << FExpBase::TOWER_LEVEL));
 
+		// TODO: Add check for F, FSub, VALUES_PER_ROW
```

Comment:
This is just duplicating the comment as above. I may fix both of those as follow ups.